### DISCUSSION TITLE
Game Crashes after pasting the contents into the /{Gameroot}/GameName/Binaries/Win64/ folder.

### DISCUSSION THREAD

**Discussion by AbyssalBeasts** [UE4SS-RE/RE-UE4SS#395] [General]
Game Crashes when I try to run the world.
Crash report says the following:

Unhandled Exception: EXCEPTION_ACCESS_VIOLATION reading address 0x000001ec509e3058
"
UE4SS
UE4SS
UE4SS
UE4SS
Palworld_Win64_Shipping
Palworld_Win64_Shipping
Palworld_Win64_Shipping
Palworld_Win64_Shipping
UE4SS
Palworld_Win64_Shipping
Palworld_Win64_Shipping
Palworld_Win64_Shipping
UE4SS
Palworld_Win64_Shipping
Palworld_Win64_Shipping
Palworld_Win64_Shipping
Palworld_Win64_Shipping
Palworld_Win64_Shipping
Palworld_Win64_Shipping
Palworld_Win64_Shipping
UE4SS
Palworld_Win64_Shipping
Palworld_Win64_Shipping
Palworld_Win64_Shipping
Palworld_Win64_Shipping
Palworld_Win64_Shipping
Palworld_Win64_Shipping
Palworld_Win64_Shipping
Palworld_Win64_Shipping
Palworld_Win64_Shipping
kernel32
ntdll
"

Any ideas how to fix this ?

**Comment by UE4SS** [comment]
Please create an issue, and follow the template when doing so.
